released/packages/coq-itauto.8.13/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "frederic.besson@inria.fr"
+homepage: "https://gitlab.inria.fr/fbesson/itauto"
+dev-repo: "git@gitlab.inria.fr:fbesson/itauto.git"
+authors: ["Frédéric Besson"]
+bug-reports: "frederic.besson@inria.fr"
+license: "GPL 3"
+synopsis: "'itauto' is a reflexive SAT solver parameterised by a leaf tactic"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/Cdcl"]
```

Comment:
Not needed with opam 2 (probably the first line should specify `opam-version: "2.0"` too unless compatibility with opam 1 is needed). Most packages are requiring opam 2 now.

Reply:
Yes. I ran opam lint but forgot to commit.